pkg/aws/policies.go

```diff
@@ -909,6 +950,17 @@ func (c *awsClient) GetAttachedPolicy(role *string) ([]PolicyDetail, error) {
 	return policies, nil
 }
 
+func (c *awsClient) getAttachedOperatorRolePolicies(role *string) ([]*iam.AttachedPolicy, error) {
```

Comment:
Maybe we can reuse the `GetPolicies` function instead of this new function,
And then we can remove `deleteOperatorRolePolicies` and use `deletePolicies` instead, WDYT?

Reply:
Simple enough of a change, thanks for the suggestion